### DISCUSSION TITLE
Chinese symbols interfere with match bracket functionality

### DISCUSSION THREAD

**Discussion by Withsan** [helix-editor/helix#10390] [Troubleshooting]
iam writing md file in Chinese，when i was try to use `Goto matching bracket` in '（这是一段中文）' content，helix do noting。
helix version is: 24.3 
marksman version is : 1.0.0-f124c56+f124c5616cf38241e1a5e58808ed78e2d9a1d385

**Comment by woojiq** [comment]
~If this is your only complaint, try https://github.com/helix-editor/helix/pull/8294, `mm` works for me there.~
Nope.
UPD: Looks like `find_pair` function doesn't work as expected.

**Comment by pascalkuthe** [comment]
I cant reproduce this putting `(这是一段中文)` in a `foo.md` file work as expected (also tried rust and it works there too). It only doesn't work in plaintext files but that is expected/intentional behavior

**Reply by woojiq** [reply]
You need to check with Chinese brackets: `（这是一段中文）`. And the problem is that `mim` and `mam` works, but `mm` does not.
https://github.com/helix-editor/helix/blob/081f7d0bd8f3b59857be3fa6bcfccdaee31eaf69/helix-core/src/surround.rs#L6-L14
